README.md

```diff
@@ -23,6 +23,13 @@ You can install the extension from one of the following links
 
 ### Building the code
 
+#### Prerequisites
+
+Please ensure that you have at least the minimum recommended versions
+
+-   Node - 10.15.0 (Check by running `node --version`)
```

Comment:
Is there a reason to fix this to a particular node-version. In my assumption, something above Node v8 should be enough!

Reply:
I ran it with 8.11 and got snapshot errors during `npm run precheckin`. @peterdur recommended 10.15.0 since that is what we use in our build systems.

Reply:
I guess my question is: Will it not work on any other higher version of node? I mean, the verbiage should be that `Node version > 10 ` should be enough for the externals to run our code.

Reply:
The document does say "Please ensure that you have **at least** the minimum recommended versions" which implies anything above it should be okay

Reply:
Cool.